### PR TITLE
Feature/reg c2

### PR DESCRIPTION
**Pull request by fanglinfang**
https://docs.google.com/document/d/1pvVOGBIhMuiPsd27-nfNCIQlBlsE94Ybafp6Ffr9MIc/edit#
Personal build verify with 
javerage, 2014-02-10
jpce, jinter 2013-05-10

### CODE REVIEW

**Comment by fanglinfang** [comment]
Let me make one more change

File: myuw/templates/handlebars/card/registration/reg_resources.html [hunk 1]
```diff
@@ -82,15 +82,15 @@ <h4 class="sr-only">Registration resources</h4>
             </li>
         {{ /if }}
 
-	{{ #if is_pce }}
+	{{ #if is_c2 }}
```
Comment (hlee2) [inline myuw/templates/handlebars/card/registration/reg_resources.html]: PCE Time schedule should be displayed for C2 student. Place the PCE Time schedule at top of the resources link.

File: myuw/templates/handlebars/card/registration/reg_resources.html [hunk 2]
```diff
@@ -39,7 +39,7 @@ <h4 class="sr-only">Registration resources</h4>
             {{/if}}
             <li>
                 <a target="_blank"
-                   href="https://uwstudent.washington.edu/student/myplan/mplogin/netid?rd=/student/myplan/">MyPlan</a>
+                   href="https://uwstudent.washington.edu/student/myplan/mplogin/netid?rd=/student/myplan/">MyPlan</a> <span>(Non-PCE courses only)</span>
```
Comment (hlee2) [inline myuw/templates/handlebars/card/registration/reg_resources.html]: Make the '(Non-PCE courses only)' inside </a> to be consistent with other link styles in MyUW.

**Comment by hlee2** [comment]
Can we update the Regi card resource link order to 
- MyPlan (Non-PCE courses only)
- Seattle Time Schedule
- PCE Time Schedule 
- How to register PCE courses 
- Audit your degree (DARS)

File: myuw/templates/handlebars/card/registration/reg_resources.html [hunk 3]
```diff
@@ -19,7 +19,7 @@ <h4 class="sr-only">Registration resources</h4>
                     {{/unless}}
```
Comment (hlee2) [inline myuw/templates/handlebars/card/registration/reg_resources.html]: line 18: ..... {{plan_data.registration_href}}">Register using MyPlan</a> also needs '(non-PCE...)' text.

**Pull request merged**